app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -741,7 +751,23 @@ public void toggleMoreResultsBox() {
 
     @OnClick({R.id.inputAppraisalExpandBox})
     public void toggleAppraisalBox() {
-        toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        boolean visible = toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        moveInfoWindow(visible);
+    }
+
+    /**
+     * Move infoLayout window to top/bottom.
+     * @param toTop
+     */
+    private void moveInfoWindow(boolean toTop) {
+        if (toTop) {
+            layoutParams.gravity = Gravity.CENTER | Gravity.TOP;
+            Log.d(TAG, "moveInfoWindow: top");
+        } else {
+            layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
+            Log.d(TAG, "moveInfoWindow: bottom");
+        }
+        windowManager.updateViewLayout(infoLayout, layoutParams);
```

Comment:
What do you think of 7f3fdcad402cbc9ab92787a70ed7a24baf8a6f52? That might need further refinement. @DJCrapsody @nahojjjen and @imapokemon4 what do you think? (I'd probably at least go for capitals).

> But you would need to preserve the first option in each group. This need to be something meaningful (as there are no labels).

I've kept your first label there. Having something meaningful could suggest longer phrases.